fcrepo-http-api/src/main/resources/views/common.js

```diff
@@ -94,7 +94,15 @@
       const url = this.href;
       http('HEAD', url, function(res) {
         if (res.status >= 400 || res.getResponseHeader('Link') == null) {
-          location.href = url;
+          var newLocation = url;
```

Comment:
Ideally, we would be able to more directly detect that a redirect took place, such as comparing the original `url` to the response `Location`, or check for a `307` status.
But in the absence of that possibility, this looks good.
@acoburn : thoughts?